<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataDownloadPage</class>
 <widget class="QWidget" name="DataDownloadPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>406</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="screenLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>СКРИНШОТЫ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dontDownloadScreensRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Не скачивать</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadLastScreenRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Последний</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadAllScreensRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Все</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSvgWidget" name="separatorSVG_1" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="averageICPLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>СРЕДНЕЕ ВЧД</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dontDownloadAverageICPRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Не скачивать</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadLast24HrsAvreageICPRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>За последние
24 часа</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadAllAverageICPRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Все</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSvgWidget" name="separatorSVG_2" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="sensorReadingsLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>30</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>ПОКАЗАНИЯ
ДАТЧИКА</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dontDownloadSensorReadingsLabelRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>Не скачивать</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadLastrHourSensorReadingsLabelRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>За последний
час</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="downloadLast24HrsSensorReadingsLabelRadioButton">
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton::indicator {
    width: 100px;
    height: 50px;
}

QRadioButton{
    spacing: 6px;
	color: white;
}

QRadioButton::indicator:checked {
    image: url(:/icons/radioButtonSelected.svg);
}

QRadioButton::indicator:unchecked
  {
    image: url(:/icons/radioButtonUnselected.svg);
  }</string>
         </property>
         <property name="text">
          <string>За последние
24 часа</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>qsvgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
